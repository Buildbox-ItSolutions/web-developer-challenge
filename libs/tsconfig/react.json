{
  "$schema": "https://json.schemastore.org/tsconfig",
  "display": "React",
  "extends": "./base.json",
  "compilerOptions": {
    "jsx": "react-jsx",
    "skipLibCheck": true,
    "lib": ["ES2015", "DOM"],
    "module": "ESNext",
    "target": "es6"
  }
}
