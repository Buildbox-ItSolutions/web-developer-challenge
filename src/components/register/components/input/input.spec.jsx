import { render, screen } from '@testing-library/react';
import { Input } from '.';

describe('<Input/>', () => {
  it('should render Input', () => {
    render(<Input />);

    const input = screen.getByPlaceholderText(/digite/i);

    expect(input).toBeInTheDocument();
  });

  it('should match snapshot', () => {
    const { container } = render(<Input disabled={true} />);

    expect(container).toMatchSnapshot();
  });
});
