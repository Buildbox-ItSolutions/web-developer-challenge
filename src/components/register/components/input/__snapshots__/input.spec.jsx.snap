// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input/> should match snapshot 1`] = `
<div>
  <input
    id="input-name"
    placeholder="Digite seu nome"
    type="text"
  />
</div>
`;
